src/status_im2/common/standard_auth/keycard/style.cljs

```diff
@@ -0,0 +1,62 @@
+(ns status-im2.common.standard-auth.keycard.style
+  (:require
+    [quo2.foundations.colors :as colors]))
+
+(def container
+  {:flex 1})
+
+(def inner-container
+  {:flex               1
+   :padding-horizontal 20})
+
+(def context-tag
+  {:margin-top     12
+   :margin-bottom  56
+   :flex-direction :row})
+
+(def close-button
+  {:margin-bottom 12})
+
+(def secondary-text
+  {:margin-top    8
+   :margin-bottom 20})
+
+(def try-again-button
+  {:flex               1
+   :padding-horizontal 20})
+
+(def keycard
+  {:margin-bottom 12})
+
+(def divider
+  {:margin-top 20})
+
+(def reset-keycard-button
+  {:align-self
+   :flex-start
+   :margin-top 12})
+
+(defn get-digits-container-style
+  [max-attempt-reached]
+  {:flex               1
+   :padding-horizontal 60
+   :align-items        :center
+   :flex-direction     :row
+   :justify-content    :space-between
+   :margin-bottom      (if max-attempt-reached 0 34)})
+
+(defn digit-style
```

Comment:
The `style` suffix is redundant in the style namespace, unless you want to avoid shadowing something, but that's not the case it seems. Same for the function `get-digits-container-style`.

Reply:
@ilmotta Makes sense. Removed the `style` suffix.